corn-field-il.md

```diff
@@ -0,0 +1,7 @@
+## Corn field :corn:
```

Comment:
Unfortunately, there are no hotels in my town/cornfield.

I suggest staying somewhere near the U of I campus in Champaign/Urbana! :hotel: